packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [18.2.0]
+
+### Added
+
+- Add new event `AccountsController:accountRenamed` ([#4664](https://github.com/MetaMask/core/pull/4664)), ([#4660](https://github.com/MetaMask/core/pull/4660))
+- Add new `nameLastUpdatedAt` timestamp to account's metadata ([#4589](https://github.com/MetaMask/core/pull/4589))
```

Comment:
Might be a bit too much? I just feel like we are adding a new field without too much explanations (even though, it's pretty self-explanatory!)

Lemme know what you think!

```suggestion
- Add new `nameLastUpdatedAt` timestamp to account's metadata ([#4589](https://github.com/MetaMask/core/pull/4589))
  - It will be updated every time an account gets renamed through `setAccountName` or `updateAccountMetadata`.
```